tests/meta/clean/test_clean_flow_cell_run_directories.py

```diff
@@ -0,0 +1,100 @@
+"""Tests for cleaning flow cell run directories using
+cg.meta.clean.flow_cell_run_directories.RunDirFlowCell """
+from unittest import mock
+
+from cg.constants import HousekeeperTags
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+from cg.meta.clean.flow_cell_run_directories import RunDirFlowCell
+
+
+@mock.patch("cg.apps.housekeeper.hk.HousekeeperAPI")
+@mock.patch("cg.store.Store")
+def test_age(
+    mock_statusdb,
+    mock_hk,
+    flowcell_path,
+    timestamp_yesterday,
+):
+    # GIVEN a flow cell with a sequenced date:
+    flow_cell_obj = RunDirFlowCell(flowcell_path, mock_statusdb, mock_hk)
+    flow_cell_obj._sequenced_date = timestamp_yesterday
+
+    # WHEN determining the age of a flow cell
+    result = flow_cell_obj.age
+
+    # THEN the age property of the flow cell should be set to 1 day
+    assert result == 1
+
+
+@mock.patch("cg.apps.housekeeper.hk.HousekeeperAPI")
+@mock.patch("cg.store.Store")
+def test_sequenced_date_from_statusdb(
+    mock_statusdb,
+    mock_hk,
+    flowcell_path,
+    timestamp_yesterday,
+):
+    # GIVEN a flow cell with a sequenced_at date in statusdb
+    flow_cell_obj = RunDirFlowCell(flowcell_path, mock_statusdb, mock_hk)
+    mock_statusdb.flowcell.return_value.sequenced_at = timestamp_yesterday
+
+    # WHEN determining the age of a flow cell
+    result = flow_cell_obj.sequenced_date
+
+    # THEN the sequenced_date property of the flow cell should be set to the sequenced_at date in
+    # statusdb
+    assert result == timestamp_yesterday
+
+
+@mock.patch("cg.apps.housekeeper.hk.HousekeeperAPI")
+@mock.patch("cg.store.Store")
+def test_sequenced_date_from_run_name(
+    mock_statusdb,
+    mock_hk,
+    flowcell_path,
+):
+    # GIVEN a flow cell that does not exist in statusdb
+    flow_cell_obj = RunDirFlowCell(flowcell_path, mock_statusdb, mock_hk)
+    mock_statusdb.flowcell.return_value = None
+
+    # WHEN determining the age of a flow cell
+    result = flow_cell_obj.sequenced_date
+
+    # THEN the sequenced_date property of the flow cell should be derived from the run name of
+    # the flow cell
+    assert result == flow_cell_obj.derived_date
+
+
+@mock.patch("cg.apps.housekeeper.hk.HousekeeperAPI")
+@mock.patch("cg.store.Store")
+def test_archive_sample_sheet_no_bundle(mock_statusdb, mock_hk, flowcell_path, novaseq_dir):
+    # GIVEN a flow cell
+    flow_cell_obj = RunDirFlowCell(flowcell_path, mock_statusdb, mock_hk)
+    # that has a sample sheet
```

Comment:
```suggestion
    # GIVEN a sample sheet connected to the flow cell
```
I think is is more readable to stick to multiple GIVEN in the `#GIVEN-WHEN-THEN` instead of just comments

Reply:
fixed in https://github.com/Clinical-Genomics/cg/pull/1371/commits/b8ef51695f0716a0c8ae67b010f094150e8e20b0